### PR TITLE
Deprecate the module

### PR DESCRIPTION
**Pull request by smortex**

### CODE REVIEW

**Comment by alexjfisher** [comment]
@vbatoufflet Hi!  We're not actually sure we've got any users, but we're about to deprecate this module as there are no PPA packages for anything other than Ubuntu 14.04.  Have you any interest in us keeping this puppet module going (perhaps you know of people in your community who are or have used it in the past)?  If so, would you be able to publish new PPAs?

Thanks!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Facette
 
+:warning: This module is declared as deprecated by Vox Pupuli, see #56. :warning:
```
Comment (ekohl) [inline README.md:3]: I'd make this a link. Something like
```suggestion
:warning: This module is declared as deprecated by Vox Pupuli, see [#56](https://github.com/voxpupuli/puppet-facette/issues/56). :warning:
```

Comment (smortex) [reply README.md]: lol, I expected GitHub to do this for me :P

**Comment by vbatoufflet** [comment]
Hi @alexjfisher,

Sorry for the late response. It's been a while since I didn't work on the current version of Facette's code base. The future one has been fully rewritten but it's not yet ready and will introduce incompatibilities for sure.

I think it's safe to deprecate this module :)

**Pull request merged**